front/src/components/UI/Header/header.tsx

```diff
@@ -0,0 +1,9 @@
+import { HeaderWrapper } from "./header.styled"
+import styled from "styled-components";
+
+const Header = styled(HeaderWrapper)`
```

Comment:
Ogólnie ten komponent i ogólnie plik nie są potrzebne. W poprzednim komentarzu napisałem że można zaimportować gdzieś bezpośrednio z pliku styled.ts. Style które są w 2 poniższych linijkach możesz dopisać bezpośrednio w tamtym komponencie - o ile są potrzebne. Inherit może być domyślnym zachowaniem więc jeśli nie powoduje dana reguła żadnych efektów to nie warto jej w ogóle pisać.